pkg/controllers/user/logging/utils/kafka.go

```diff
@@ -134,3 +135,11 @@ func (w *kafkaTestWrap) kafkaConn(dial dialer.Dialer, config *tls.Config, smartH
 
 	return kafka.NewConn(LeaderConn, w.Topic, defaultPartition), nil
 }
+
+func wrapErrEOF(err error) error {
+	if err == io.EOF {
+		return errors.New("unexpected EOF, connect is closed by remote server")
```

Comment:
connection, not connect

Reply:
and the tense isn't quite right, dropping "is" might make it sound better
"connection closed by remote server"